src/entryPoints/hyperswitch/AcceptInviteHome.res

```diff
@@ -0,0 +1,147 @@
+@react.component
+let make = () => {
+  open APIUtils
+  let updateDetails = useUpdateMethod()
+  let showToast = ToastState.useShowToast()
+  let (showModal, setShowModal) = React.useState(_ => false)
+  let switchMerchantListValue = Recoil.useRecoilValueFromAtom(
+    HyperswitchAtom.switchMerchantListAtom,
+  )
+  let (merchantListValue, setMerchantListValue) = React.useState(_ => [])
+  let (acceptedMerchantId, setAcceptedMerchantId) = React.useState(_ => [])
+
+  React.useEffect1(() => {
+    let filteredSwitchMerchantList = switchMerchantListValue->Array.filter(ele => !ele.is_active)
+    setMerchantListValue(_ => filteredSwitchMerchantList)
+    let arr = Array.make(~length=filteredSwitchMerchantList->Array.length, false)
+    setAcceptedMerchantId(_ => arr)
+    None
+  }, [switchMerchantListValue])
```

Comment:
can you please check the length of switchMerchantListValue

Reply:
Updated